From: Jonas Karlsson <jonas.karlsson@qt.io>
Subject: [PATCH] PhysX: Do not include the vehicle library

The vehicle library is not part of our copy of PhysX, since Qt Quick 3D
Physics exposes no vehicle types, so PxPhysicsAPI.h must not include it.
---
 src/3rdparty/PhysX/include/PxPhysicsAPI.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/3rdparty/PhysX/include/PxPhysicsAPI.h b/src/3rdparty/PhysX/include/PxPhysicsAPI.h
--- a/src/3rdparty/PhysX/include/PxPhysicsAPI.h
+++ b/src/3rdparty/PhysX/include/PxPhysicsAPI.h
@@ -210,9 +210,6 @@ Alternatively, one can instead directly #include a subset of the below files.
 #include "extensions/PxSerialization.h"
 #include "extensions/PxRepXSerializer.h"
 
-//Vehicle Simulation
-#include "vehicle/PxVehicleAPI.h"
-
 //Connecting the SDK to Visual Debugger
 #include "pvd/PxPvdSceneClient.h"
 #include "pvd/PxPvd.h"
